### DISCUSSION TITLE
Multi pods on k8s

### DISCUSSION THREAD

**Discussion by yuval987** [jellyfin/jellyfin#9184] [Q&A]
Hi,
I installed the app on k8s and I have an error when I trying to add pod, the error came from the DB:

Busy: SQLitePCL.pretty.SQLiteException: database is locked
   at SQLitePCL.pretty.SQLiteException.Throw(Int32 rc, Int32 extended, String msg)
   at SQLitePCL.pretty.SQLiteException.CheckOk(sqlite3 db, Int32 rc)

Is it possible to deploy this app with scaling?
Maybe can I connect to remote DB? Like MySQL?

Thanks

**Comment by Shadowghost** [comment]
Currently this is not possible.

**Reply by yuval987** [reply]
About what?
Using remote DB?
And maybe do you know if they working on this?

**Reply by Shadowghost** [reply]
Jellyfin currently only works with a local SQLite database, so replication and multiple (write) access on the DB is not possible.

Rewriting the DB to an ORM is still an ongoing process. Once everything is migrated, support for external database systems like MariaDB or PostgreSQL is possible.

**Reply by yuval987** [reply]
Thanks!

**Reply by yuval987** [reply]
Are there any updates?

**Reply by Serverfrog** [reply]
Are there any updates regarding the Rewrite to ORM?, due to the fact that the application is restarted not gracefully sometimes, the sqlite Database is faster corrupt then most politicians
